src/openrct2/object/ObjectList.cpp

```diff
@@ -64,7 +64,7 @@ int32_t object_entry_group_encoding[] = {
 ObjectList::const_iterator::const_iterator(const ObjectList* parent, bool end)
 {
     _parent = parent;
-    _subList = _parent->_subLists.size();
+    _subList = end ? _parent->_subLists.size() : 0;
```

Comment:
What is this change for?

Reply:
I can't remember anymore. I'll have to check